src/modules/user/user.controller.ts

```diff
@@ -99,4 +101,26 @@ export class UserController {
   async deleteReview(@Param('id', ParseUUIDPipe) id: string): Promise<void> {
     return await this.reviewService.deleteReview(id);
   }
+
+  @ApiBearerAuth()
+  @UseGuards(JwtAuthGuard)
+  @Post(ERouteName.SAVE_GENERAL_INFO)
+  async saveGeneralInfo(
+    @Param('id', ParseUUIDPipe) id: string,
+    @Body() generalInfoDto: GeneralInfoDto,
+  ): Promise<void> {
+    return await this.userService.saveGeneralInfo({ id, generalInfoDto });
```

Comment:
why pass object? mb remove object and then You can remove object in service input?